tc/lang/sema.h

```diff
@@ -627,10 +677,12 @@ struct Sema {
     }
     return it == the_env.end() ? nullptr : it->second;
   }
-  TreeRef c(int kind, const SourceRange& range, TreeList&& trees) {
+
+  TreeRef createCompound(int kind, const SourceRange& range, TreeList&& trees) {
```

Comment:
We have been using ```makeXYZ``` syntax throughout the codebase for functions that return an owning pointer, let's try to be consistent.

Reply:
it was named `create...` because that's how the functions are named in lexer/sema/parser codebade to create a new node like: https://github.com/facebookresearch/TensorComprehensions/blob/master/tc/lang/tree_views.h#L204. The coding conventions recommend to have consistency locally within lexer/sema/parser (for example) code base here https://github.com/facebookresearch/TensorComprehensions/blob/master/CodingConventions.md#naming

if the naming should be `make...`, 1. can you kindly create a rule in `CodingConventions.md` so that people can follow that? 2. it would be also nice to define when should global/local rules be followed? thanks :)

Reply:
sounds good, local trump global :)